staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -754,6 +752,22 @@ func (s *store) ttlOpts(ctx context.Context, ttl int64) ([]clientv3.OpOption, er
 	return []clientv3.OpOption{clientv3.WithLease(id)}, nil
 }
 
+// ensureMinimumResourceVersion returns an error unless provided actualRevision from etcd is greater
+// than or equal to minimumResourceVersion.
+func (s *store) ensureMinimumResourceVersion(minimumResourceVersion string, actualRevision int64) error {
+	if minimumResourceVersion != "" {
+		fromRV, err := s.versioner.ParseResourceVersion(minimumResourceVersion)
+		if err != nil {
+			return apierrors.NewBadRequest(fmt.Sprintf("invalid resource version: %v", err))
+		}
+		if actualRevision < int64(fromRV) {
```

Comment:
Just double checking - shouldn't this be "<=" ?
Can you add a comment about this (no matter what the outcome is)?

Reply:
I'll add a comment. I believe `<` is correct (we use it here too: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/storage/cacher/watch_cache.go#L301).

Reply:
let's triple check this logic, this is the type of thing that makes for compatibility headaches if it ends up wrong for a release. if we make this `actualRevision <= int64(fromRV)` or `actualRevision < int64(fromRV) - 1`, do existing tests catch that and fail?

Reply:
Certainly. I've had to squint at this a couple times to make sure I believe it's right.

If I change the current conditional to `if minimumRV -1 > actualRevision` or `if minimumRV > actualRevision -1`, tests will fail, e.g.:
```
--- FAIL: TestList (0.09s)                                                                                                                                                                                                                                       
 store_test.go:1158: (test List on existing key with minimum resource version set to current resource version): List failed: Timeout: Too large resource version: 6, current: 6                                                                               

--- FAIL: TestGet/latest_resource_version (0.00s)
 store_test.go:262: Get failed: Timeout: Too large resource version: 3, current: 3
```